.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: go
 go:
   - 1.7
+  - 1.8
+  - 1.9
 install:
   - go get -v github.com/Masterminds/glide
   - glide install
```

Comment:
while you're here, could you change these to use the appropriate targets from the Makefile